validator.py

```diff
@@ -626,6 +630,83 @@ def does_key_have_value(self, key, value, yaml):
         # if we made it to here, we found the key - check the value!
         return data == value
 
+
+    def scenes_with_transitions(self):
+        '''
+        Looks through the yaml file to make sure that every scene from 0 to n-1 has 
+        a transitions field
+        '''
+        data = copy.deepcopy(self.loaded_yaml)
+        scenes = data['scenes']
+        for i in range(0, len(scenes)-1):
+            if 'transitions' not in scenes[i]:
+                self.logger.log(LogLevel.WARN, "Key 'transitions'  must be provided within each entry in 'scenes' but is missing at scenes[" + str(i) + "]")
```

Comment:
The final scene (e.g., `scenes[n-1]`) doesn't require a transition.  So, "must be provided within all but the last entry in 'scenes' [...]"

Reply:
Changed